src/controllers/controllerengine.cpp

```diff
@@ -390,6 +394,12 @@ bool ControllerEngine::internalExecute(QScriptValue thisObject, QScriptValue fun
         return false;
     }
 
+    if (functionObject.isError()) {
+        qDebug() << "ControllerEngine::internalExecute:"
+                 << functionObject.toString();
```

Comment:
It would be easy to call scriptErrorDialog() here, but IMO popping up a dialog repeatedly is annoying. When debugging a mapping, I'd rather look through the log than have to click to close a bunch of dialogs.